src/Datadog.Trace/SpanTypes.cs

```diff
@@ -66,5 +66,10 @@ public static class SpanTypes
         /// The span type for msbuild integration.
         /// </summary>
         public const string Build = "build";
+
+        /// <summary>
+        /// The span type for CosmosDb integration.
+        /// </summary>
+        public const string CosmosDb = "cosmosdb";
```

Comment:
As mentioned in https://github.com/DataDog/dd-trace-dotnet/pull/1473#discussion_r635485072, we should consider using the `sql` span type for this integration.

Reply:
I already moved to the sql span type, I forgot to remove this. Fixed in the latest version.